src/main/play-doc/operation/Install.md

```diff
@@ -1069,7 +938,7 @@ frontend monitor
 Run the HAProxy Docker container.
 
 ```
```

Comment:
Maybe reuse here 
'Please ensure the hostname is set correctly or substitute your addresses as appropriate for $(hostname). To set the hostname ..."
Now that this section is EC2 specific?